cmd/query/app/flags_test.go

```diff
@@ -30,10 +32,32 @@ func TestQueryBuilderFlags(t *testing.T) {
 		"--query.ui-config=some.json",
 		"--query.base-path=/jaeger",
 		"--query.port=80",
+		"--query.additional-headers=access-control-allow-origin:blerg",
+		"--query.additional-headers=whatever:thing",
 	})
-	qOpts := new(QueryOptions).InitFromViper(v)
+	qOpts := new(QueryOptions).InitFromViper(v, zap.NewNop())
 	assert.Equal(t, "/dev/null", qOpts.StaticAssets)
 	assert.Equal(t, "some.json", qOpts.UIConfig)
 	assert.Equal(t, "/jaeger", qOpts.BasePath)
 	assert.Equal(t, 80, qOpts.Port)
+	assert.Equal(t, http.Header{"Access-Control-Allow-Origin": []string{"blerg"}, "Whatever": []string{"thing"}}, qOpts.AdditionalHeaders)
```

Comment:
reads better this way:
```suggestion
	assert.Equal(t, http.Header{
	           "Access-Control-Allow-Origin": []string{"blerg"}, 
	           "Whatever": []string{"thing"}
	       }, qOpts.AdditionalHeaders)
```